UKHomeOffice/sgmr-service
# 1.1.2 SPCR-351 Remove postcode error on responsible person form

## Description
Postcode/Zipcode was mandatory on the Skipper Address page. This has now been made optional.

**Do not merge until SPCR-350 is merged**

## To Test
- Sign in and create a new voyage plan
- Go to the skipper details page
- Click save and continue
- You should get errors (but none postcode related)
- Fill out all mandatory fields (leave postcode blank)
- Click save and continue
- You should proceed to the next page

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
